src/SDKs/SqlManagement/Sql.Tests/Utilities/SqlManagementTestUtilities.cs

```diff
@@ -40,6 +40,10 @@ public class SqlManagementTestUtilities
 
         public const string DefaultEuapPrimaryLocation = "East US 2 EUAP";
 
+        public const string DefaultLogin = "dummylogin";
+
+        public const string DefaultPassword = "Un53cuRE!";
```

Comment:
It's [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines) number 2 :)